pkg/api/frontendsettings_test.go

```diff
@@ -83,7 +81,7 @@ func setupTestEnvironment(t *testing.T, cfg *setting.Cfg, features featuremgmt.F
 
 	m := web.New()
 	m.Use(getContextHandler(t, cfg).Middleware)
-	m.UseMiddleware(web.Renderer(filepath.Join(setting.StaticRootPath, "views"), "[[", "]]"))
+	m.UseMiddleware(web.Renderer(filepath.Join("", "views"), "[[", "]]"))
```

Comment:
Nit:
```suggestion
	m.UseMiddleware(web.Renderer("views", "[[", "]]"))
```

Reply:
I didn't really care since tests. Lots of these :cry:

Reply:
I originally thought maybe was kept because `filepath.Join("", "views")` resolves to `/views` perhaps - feel free to disregard for now and can tidy this up later

Reply:
I searched and replaced for now. The tests doesn't seem to care whatever we set there :shrug: